W-15818513-dw-functions-27-duke (#323)
mulesoft/docs-dataweave: modules/ROOT/pages/dw-mime.adoc

= Mime (dw::module::Mime)

This module contains helper functions for working with MIME type.

To use this module, you must import it to your DataWeave code, for example,
by adding the line `import * from dw::module::Mime` to the header of your
DataWeave script.

_Introduced in DataWeave version 2.7.0._

== Functions

[%header, cols="1,3"]
|===
| Name  | Description
| xref:dw-mime-functions-fromstring.adoc[fromString] | Transforms a MIME type string value representation to a `MimeType`.

> | xref:dw-mime-functions-ishandledby.adoc[isHandledBy] | Returns `true` if the given `MimeType` value can be handled the base `MimeType` value.

Reviewer: ```suggestion
| xref:dw-mime-functions-ishandledby.adoc[isHandledBy] | Returns `true` if the given `MimeType` value is handled by the base `MimeType` value.
```
